### PR TITLE
creating target for management plane

### PR DESCRIPTION
**Pull request by shinychimra07**
Using Skyramp's deployer to install skyramp dashboard in k8s cluster

Test case
```
SC-MacBook-Pro:management-plane sc$ skyramp deployer up dashboard -v --local-images
INFO Setting log level to Info                    
INFO Project path is                              
INFO Validate license                              module=licensing
INFO Running deployer up command with target dashboard  cmd=up
INFO Reading project directory                     module=loader
INFO Added target dashboard.yaml to project        module=loader
INFO No dependency errors                          module=loader
INFO validation passed                             module=loader
INFO Using 296613639307.dkr.ecr.us-west-2.amazonaws.com/dev/worker:70566dd4 for worker image  cmd=up
INFO builder option: &{true}                      
INFO builder option: &{true}                      
INFO Containers to bring up: []                    module=loader
INFO Containers to mock: []                        module=loader
INFO translate helm chart ../../charts/mongo-datastore  module=loader
INFO attempt ../../charts/mongo-datastore/values.yaml  module=utils
INFO processing mongo-datastore/charts/community-operator/charts/community-operator-crds/templates/mongodbcommunity.mongodb.com_mongodbcommunity.yaml  module=loader
INFO processing mongo-datastore/charts/community-operator/templates/mongodbcommunity_cr_with_tls.yaml  module=loader
INFO processing mongo-datastore/templates/mongo-statefulset.yaml  module=loader
INFO processing mongo-datastore/templates/NOTES.txt  module=loader
INFO processing mongo-datastore/charts/community-operator/templates/operator_roles.yaml  module=loader
INFO processing mongo-datastore/charts/community-operator/templates/operator.yaml  module=loader
INFO processing mongo-datastore/charts/community-operator/templates/database_roles.yaml  module=loader
INFO processing mongo-datastore/templates/tests/test-connection.yaml  module=loader
INFO processing mongo-datastore/templates/mongodb-secret.yaml  module=loader
INFO processing mongo-datastore/templates/mongo-service.yaml  module=loader
INFO Adding Secret kind                            module=builder
INFO Adding MongoDBCommunity kind                  module=builder
INFO translate helm chart ../../charts/web-server  module=loader
INFO attempt ../../charts/web-server/values.yaml   module=utils
INFO processing web-server/templates/frontend-service.yaml  module=loader
INFO processing web-server/templates/clusterrole.yaml  module=loader
INFO processing web-server/templates/NOTES.txt     module=loader
INFO processing web-server/templates/tests/test-connection.yaml  module=loader
INFO processing web-server/templates/serviceaccount.yaml  module=loader
INFO processing web-server/templates/frontend-deployment.yaml  module=loader
INFO processing web-server/templates/clusterrolebinding.yaml  module=loader
INFO processing web-server/templates/backend-service.yaml  module=loader
INFO processing web-server/templates/backend-deployment.yaml  module=loader
INFO Adding ServiceAccount kind                    module=builder
INFO Adding service dashboard-client               module=builder
INFO Service dashboard-client does not have skyramp.dev/protocol, skipping ingress creation  
INFO Adding service dashboard-server               module=builder
INFO Service dashboard-server does not have skyramp.dev/protocol, skipping ingress creation  
INFO Containers to bring up: []                    module=loader
INFO Containers to mock: []                        module=loader
INFO Translation finished                          cmd=up
INFO cluster is a kind cluster; starting image sideloading  cmd=up
INFO Sideloading image: 296613639307.dkr.ecr.us-west-2.amazonaws.com/dev/worker:70566dd4  module=kind
Loading local images:      [##############################################################] 100 %
INFO Created k8s lcm                               cmd=up
INFO Create ServiceAccounts                        module=lcm
INFO Create Roles                                  module=lcm
INFO Create RoleBindings                           module=lcm
INFO Create config maps                            module=lcm
INFO Create services                               module=lcm
INFO Create deployments                            module=lcm
INFO Create ingresses                              module=lcm
INFO Create http proxies                           module=lcm
dashboard-server-6c94b5d779-rnl4t            local image      ready
mongodb-kubernetes-operator-68fb789cdf-8sd68 local image      ready
skyramp-worker-76c584b77d-l9h7s              local image      ready
dashboard-client-849c787789-qmhcz            local image      ready
All pods are ready.
```

### CODE REVIEW

**Review by PellePedro — APPROVED** [review]
LGTM

File: charts/worker/values.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ bootConfig:
 ManagementPlaneConfig:
   mongoConfig:
     enabled: true
-    mongodbUrl: "mongodb://admin:pa$$w0rd@mongo-datastore-svc.skyramp:27017/?authSource=admin&authMechanism=SCRAM-SHA-256"
+    mongodbUrl: "mongodb://admin:pa$$word@mongo-datastore-svc.skyramp:27017/?authSource=admin&authMechanism=SCRAM-SHA-256"
```
Comment (sacheth003) [inline charts/worker/values.yaml:66]: Random question, but why was this required?

**Review by sacheth003 — APPROVED** [review]

File: charts/worker/values.yaml (hunk 1 above)
Comment (shinychimra07) [reply charts/worker/values.yaml:66]: This is required by the skyramp worker to push test results and active mocks to mongo db

Comment (sacheth003) [reply charts/worker/values.yaml:66]: Oh what I mean is where was the password change done to require this?

File: skyramp/management-plane/targets/dashboard.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+containers:
+  - type: helm
+    releaseName: skyramp-dashboard
+    path: ../../charts/mongo-datastore
+    valuesPath: ../../charts/mongo-datastore/values.yaml
+    values:
+     mongodb:
+      auth:
+        username: "admin"
+        password: "pa$$word"
```
Comment (shinychimra07) [inline skyramp/management-plane/targets/dashboard.yaml:11]: @sacheth003 I updated the password here
